src/app/components/login/login.component.ts

```diff
@@ -1,64 +1,66 @@
-import { AuthService } from './../../services/auth.service';
 import { CommonModule } from '@angular/common';
-import { Component } from '@angular/core';
-import { FormControl, FormGroup, ReactiveFormsModule, Validators, FormBuilder } from '@angular/forms';
+import { Component, OnInit } from '@angular/core';
+import {
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
 import { LoginRequest } from '../../login-request';
-import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { loginUser } from '../store/actions/login.actions';
+import { InlineNotificationService } from '../../services/inline-notification.service';
+import { InlineNotification } from '../../inlineNotification';
+import { ValidationMessagesComponent } from '../validation-messages/validation-messages.component';
 
 @Component({
   selector: 'app-login',
   standalone: true,
-  imports: [ReactiveFormsModule, CommonModule],
+  imports: [ReactiveFormsModule, CommonModule, ValidationMessagesComponent],
   templateUrl: './login.component.html',
-  styleUrl: './login.component.scss'
+  styleUrl: './login.component.scss',
 })
-export class LoginComponent {
-  email : FormControl<string | null> = new FormControl<string>('', [Validators.required, Validators.email]) 
-  password : FormControl<string | null> = new FormControl<string>('', [Validators.required, Validators.minLength(5)]) 
-  
-  loginForm!: FormGroup;
+export class LoginComponent implements OnInit {
+  inlineNotification: InlineNotification = { show: false, type: '', text: '' };
 
-  inlineNotification: {show: boolean; type: string; text: string} = {
-    show: false,
-    type: '',
-    text: ''
-  }
-  constructor(private FormBuilder: FormBuilder, private authService: AuthService, private router: Router) {
-     this.loginForm = this.FormBuilder.group({
-      email: this.email,
-      password: this.password
-     })
+  ngOnInit(): void {
+    this.notificationService.notification$.subscribe((notification) => {
+      if (notification) {
+        this.inlineNotification = notification;
+      }
+    });
```

Comment:
unsubscribe after you subscribe to the observable to avoid memory leaks

Reply:
done